<launch>
    <arg name="debug" default="false"/>
    <arg name="use_own_params" default="true"/>
    <arg name="robot_name" default="biba"/>

    <group if="$(arg use_own_params)">
        <rosparam file="$(find bigbang_eurobot)/config/$(arg robot_name)/interconnect.yaml"/>
    </group>

    <node pkg="bigbang_eurobot" type="interconnect.py" name="interconnect" output="screen" respawn="true" respawn_delay="1"> 
        <param name = "debug" value = "$(arg debug)"/>
    </node>

</launch>